data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json

```
@@ -1,4 +1,301 @@
+      "ter_set": "t_dirt",
+      "items": [
+        { "item": "withered", "prob": 50, "count": [ 1, 2 ] },
+        { "item": "leaves", "count": [ 1, 10 ] },
+        { "item": "twig", "prob": 80, "count": [ 1, 5 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_ceiba",
+    "name": "ceiba",
+    "description": "A massive tropical tree belonging to the 'Ceiba' genus.  These gigantic trees can be found from Mexico to Southern Chile.",
+    "symbol": "7",
+    "color": [ "green", "green", "green", "green" ],
+    "move_cost": 0,
+    "coverage": 80,
+    "flags": [ "FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+    "bash": {
+      "str_min": 80,
+      "str_max": 180,
+      "sound": "crunch!",
+      "sound_fail": "whack!",
+      "ter_set": "t_dirt",
+      "items": [
+        { "item": "stick_long", "count": [ 3, 10 ] },
+        { "item": "splinter", "count": [ 10, 25 ] },
+        { "item": "leaves", "count": [ 50, 150 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_acacia",
+    "name": "acacia",
+    "description": "A slender subtropical tree belonging to the 'Acacia' genus.  Members of this genus are mostly constrained to Australia but have been brought abroad for multiple purposes with some being considered invasive species.",
+    "symbol": "7",
+    "color": [ "yellow", "green", "green", "green" ],
+    "move_cost": 0,
+    "coverage": 80,
+    "flags": [ "FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+    "bash": {
+      "str_min": 80,
+      "str_max": 180,
+      "sound": "crunch!",
+      "sound_fail": "whack!",
+      "ter_set": "t_dirt",
+      "items": [
+        { "item": "stick_long", "count": [ 3, 10 ] },
+        { "item": "splinter", "count": [ 10, 25 ] },
+        { "item": "leaves", "count": [ 50, 150 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_huizache",
+    "name": "huizache",
+    "description": "A slender tropical tree belonging to the 'Vachellia' genus.  This genus of plants is related to acacias but are native to the Americas with a wide distribution from Mexico to Brazil and worldwide.",
+    "symbol": "7",
+    "color": [ "green", "green", "green", "green" ],
+    "move_cost": 0,
+    "coverage": 80,
+    "flags": [ "FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+    "bash": {
+      "str_min": 80,
+      "str_max": 180,
+      "sound": "crunch!",
+      "sound_fail": "whack!",
+      "ter_set": "t_dirt",
+      "items": [
+        { "item": "stick_long", "count": [ 3, 10 ] },
+        { "item": "splinter", "count": [ 10, 25 ] },
+        { "item": "leaves", "count": [ 50, 150 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_parajubea",
+    "name": "parajubea",
+    "description": "A medium sized palm tree belonging to the 'Parajubea' genus.  There are only tree confirmed members of this genus, two of them endangered in Bolivia.",
```

```suggestion
    "description": "A medium sized palm tree belonging to the 'Parajubea' genus.  There are only three confirmed members of this genus, two of them endangered in Bolivia.",
```
*Not sure if pun or typo*